src/components/ModalConnectButton/View.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, PropsWithChildren } from "react";
+
+import Button, { ButtonProps } from "src/components/Button";
+
+type Props = {
+  theme?: ButtonProps["theme"];
+  size?: ButtonProps["size"];
+  onConnect: () => void;
+};
+
+const ConnectButton: FC<PropsWithChildren<Props>> = ({ children, onConnect, size, theme }) => {
+  return (
```

Comment:
Put the view details here, you will need to receive more props than these ones since you will render the two buttons here